### PR TITLE
Sadashiv pune

### PR DESCRIPTION
**Pull request by sadashivkulthe**
Added a file places to eat in Pune, MH. Ask me for more..

### CODE REVIEW

File: Sadashiv-Pune.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Pune, Maharashtra
+
+##Places to eat
+- Bhairavi
+- Delhi Chaat
+- Samarth
```
Comment (hectorsector) [inline Sadashiv-Pune.md]: @sadashivkulthe which of these is your favorite?

Comment (sadashivkulthe) [reply Sadashiv-Pune.md]: Delhi Chaat :smiley: :+1:

**Review by mectanner — APPROVED** [review]
Nice job.

**Comment by sadashivkulthe** [comment]
Thank you @mectanner

**Pull request merged**